pkg/machines/components/storagePools/storagePoolOverviewTab.jsx

```diff
@@ -34,9 +34,9 @@ export const StoragePoolOverviewTab = ({ storagePool, actionError, actionErrorDe
         <div className='ct-form-layout'>
             { actionError && <ExpandableNotification type='warning' text={actionError} textId={`${idPrefix}-error`} detail={actionErrorDetail} onDismiss={onActionErrorDismiss} /> }
 
-            { storagePool.path && <React.Fragment>
+            { storagePool.target && storagePool.target.path && <React.Fragment>
                 <label className='control-label' htmlFor={`${idPrefix}-path`}> {_("Path")} </label>
-                <div id={`${idPrefix}-path`}> {storagePool.path} </div>
+                <div id={`${idPrefix}-path`}> {storagePool.target.path} </div>
```

Comment:
This second commit ("Reformat parsed storagePool object to match the libvirt XML") looks fine, but it's a bit confusing. The first half looks like it should be squashed into the first commit, and the second is just some small code cleanup without behaviour change.

Reply:
Actually the second half has some behaviour change. Before we had an object like `{connectionName, name, id, type, path}` and now we have `{connectionName, name, id, type, target: {path: daslksla} }`

This later format change in the path key value causes the first half commit to parse different the storage Pool object.
So this commit is really connected. 

I hope this change makes sense. I really would like to keep the format of the libvirt XML in our representations, so whenever I want to access an object, I will just need to take a look in the libvirt XML format. Also path can be source path, target path etc, so especially in this case just the path key was confusing.